db/migrate/20220326085417_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram provided for this project, Post should reference the `author_id` which is an alias of the `users` table. Kindly see the suggestion below and implement it so that your posts table will have a reference column `author_id` instead of `user_id` as seen in the ERD.

```suggestion
  def change
    add_reference :posts, :author, foreign_key: { to_table: :users }
  end
```